src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,135 @@
+import {
```

Comment:
I'd move this to `scenes/Root/Sidebar` - It's not reusable.

Reply:
This is moved